### ISSUE TITLE
[Bug] Error in example "demo_space_mapping_uniform_flow_distribution"

### ISSUE THREAD

**Issue by bladesaber** [sblauth/cashocs#388]
**Describe the bug**
When run demo_space_mapping_uniform_flow_distribution.py, it throw a error below: 
```
skewness_array = self._quality_object.skewness(
TypeError: skewness(): incompatible function arguments. The following argument types are supported:
    1. (arg0: dolfin::Mesh) -> dolfin::MeshFunction<double>

Invoked with: <dolfin.cpp.mesh.Mesh object at 0x7f8c76bdae50>
```

**To Reproduce**
python demo_space_mapping_uniform_flow_distribution.py

**Screenshots**
![Screenshot from 2024-02-02 17-15-11](https://github.com/sblauth/cashocs/assets/31960871/21a0df02-c2dd-421d-88ba-a5c22dc8b2e0)

**System configuration**
conda install -c conda-forge cashocs

![Screenshot from 2024-02-02 17-17-13](https://github.com/sblauth/cashocs/assets/31960871/174b3890-d736-487f-92d3-be5c2e73a5ac)
![Screenshot from 2024-02-02 17-17-27](https://github.com/sblauth/cashocs/assets/31960871/9173ea7b-84ad-4eb4-b94a-78d15f2bbb4d)
![Screenshot from 2024-02-02 17-18-55](https://github.com/sblauth/cashocs/assets/31960871/4e8694a3-fb83-4c40-9c3e-007a2278435b)

Best regard and looking forward to you response.

**Comment by sblauth** [comment]
I cannot reproduce the issue (not even on a completely fresh installation / VM). 
For me, the demo runs without any problems.

Here are the results for my development machine:
![space_mapping_bug](https://github.com/sblauth/cashocs/assets/22995014/d6ea170c-2596-43bb-a98e-5379a922cdbb)

and here are the results for a completely fresh installation
![Screenshot from 2024-02-02 12-00-06](https://github.com/sblauth/cashocs/assets/22995014/f4cdfc55-f723-4a5e-aef2-f887360319f8)

Note that the second run on the new VM throws only an error since I did not install matplotlib.

I also tried running the demo (as it is on the main branch now, where some changes have happened), with the stable release 2.0.14, and it still worked for me. 

So in order to help you, I would need the entire output of `conda list` (and possibly `conda info`) for your environment. 
Also: Did you change anything in the demo file or have you tried using a different mesh?
